### ISSUE TITLE
BUG

### ISSUE THREAD

**Issue by ondohotola** [jqnatividad/qsv#928]
QSV leaves `qsv-cache` directories all over the place, is that a feature?

**Comment by jqnatividad** [comment]
Yes. That's the expected behavior of `luau` `qsv_register_lookup`.

It will cache the lookup table in the current working directory unless you set `QSV_CACHE_DIR` to a specific directory.

**Comment by ondohotola** [comment]
so I have to manually remove the empty directory afterwards?

Maybe I lack the insight/background but isn’t that unnecessary clutter?

**Comment by jqnatividad** [comment]
Hi @ondohotola ,
I guess you're not using `qsv_register_lookup`, right?  It's only supposed to create `qsv_cache` directories when it's needed, so that's a bug...

For now, the workaround is to set the env var `QSV_CACHE_DIR` to "/tmp" so you don't have to cleanup the directories manually.
